clients/admin-ui/src/types/api/models/PrivacyNoticeResponse.ts

```diff
@@ -16,8 +16,8 @@ export type PrivacyNoticeResponse = {
   description?: string;
   internal_description?: string;
   origin?: string;
-  regions?: Array<PrivacyNoticeRegion>;
-  consent_mechanism?: ConsentMechanism;
+  regions: Array<PrivacyNoticeRegion>;
+  consent_mechanism: ConsentMechanism;
```

Comment:
hmmm we shouldn't be modifying this file directly, since the next time we try to autogenerate types, it'll get overwritten. that said, these fields are _probably_ required, not optional, but the backend [does a workaround](https://github.com/ethyca/fides/blob/kt-privacy-notice-table-updates/src/fides/api/schemas/privacy_notice.py/#L20-L21) which I guess ends up making these fields optional. 

the proper solution would probably be for the backend to return the types properly, but in lieu of that, we should probably try to handle the optional cases (even though we know they won't be optional 🥲 )